tests/telemetry_test.py

```diff
@@ -2146,7 +2146,8 @@ def test_stores_default_nodes_stats(self, metrics_store_put_doc):
         cfg = create_config()
         metrics_store = metrics.EsMetricsStore(cfg)
         node_name = [self.node_stats_response["nodes"][node]["name"] for node in self.node_stats_response["nodes"]][0]
-        metrics_store_meta_data = {"cluster": "remote", "node_name": node_name}
+        roles = [self.node_stats_response["nodes"][node]["roles"] for node in self.node_stats_response["nodes"]][0]
+        metrics_store_meta_data = {"cluster": "remote", "node_name": node_name, "roles": roles}
```

Comment:
Would you mind moving this line after the call to recorder.record() to clarify that it does not affect the call but is just expected data? (This applies to the last three snippets too.)

Reply:
No problem! Address in https://github.com/elastic/rally/pull/1654/commits/9c64b3ddc82ebc8c09bed05481290da2820009a3.